tests/framework/validators/ExistValidatorTest.php

```
@@ -295,6 +295,68 @@ public function testSecondTargetAttributeWithError(): void
+        $val->validateAttribute($m, 'id');
+        $this->assertFalse($m->hasErrors());
+
+        $val = new ExistValidator(['targetRelation' => 'references', 'filter' => ['a_field' => 'non-existing']]);
+        $val->validateAttribute($m, 'id');
+        $this->assertTrue($m->hasErrors());
+    }
+
+    public function testPrepareConditionsWithAllowArrayAndArrayTargetAttribute(): void
+    {
+        $val = new ExistValidator(['allowArray' => true, 'targetAttribute' => ['id', 'name']]);
+        $m = new ValidatorTestMainModel();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $m. Configured minimum length is 3. (undefined)

(ShortVariable)